extensions/ARM/SPV_ARM_graph.asciidoc

```diff
@@ -0,0 +1,345 @@
+SPV_ARM_graph
+=============
+
+Name Strings
+------------
+
+SPV_ARM_graph
+
+Contact
+-------
+
+To report problems with this extension, please open a new issue at:
+
+https://github.com/KhronosGroup/SPIRV-Headers
+
+Contributors
+------------
+
+- Kevin Petit, Arm Ltd. +
+
+Notice
+------
+
+Copyright (c) 2022-2025 Arm Ltd.
+
+Status
+------
+
+Complete.
+
+Version
+-------
+
+[width="40%",cols="25,25"]
+|========================================
+| Last Modified Date | 2025-06-11
+| Revision           | 1
+|========================================
+
+Dependencies
+------------
+
+This extension is written against the SPIR-V Specification,
+Version 1.6 Revision 2.
+
+This extension requires SPIR-V 1.6.
```

Comment:
```suggestion
This extension requires SPIR-V 1.0.
```

nit, I don't see anything that would require 1.6

Reply:
I can't think of anything either. No objection to lowering the requirement to 1.0.